paddle/fluid/pybind/eager_functions.cc

```diff
@@ -958,11 +959,11 @@ PyMethodDef variable_functions[] = {
      NULL},
     /**sparse functions**/
     {"sparse_coo_tensor",
-     (PyCFunction)(void (*)(void))eager_api_sparse_coo_tensor,
+     (PyCFunctionWithKeywords)(void (*)(void))eager_api_sparse_coo_tensor,
      METH_VARARGS | METH_KEYWORDS,
      NULL},
     {"sparse_csr_tensor",
-     (PyCFunction)(void (*)(void))eager_api_sparse_csr_tensor,
+     (PyCFunctionWithKeywords)(void (*)(void))eager_api_sparse_csr_tensor,
```

Comment:
https://docs.python.org/3.10/c-api/structures.html?highlight=pycfunction#implementing-functions-and-methods   

<img width="708" alt="image" src="https://user-images.githubusercontent.com/31559413/189046807-93079d61-85de-4696-98d8-3f17dd5b6dd9.png">

According to the signature of eager_api_sparse_coo_tensor  and  eager_api_sparse_csr_tensor,  it seems more suitable  to  use  PyCFunctionWithKeywords

Reply:
In CPython source code,  in  the file  Include\methodobject.h  

<img width="875" alt="image" src="https://user-images.githubusercontent.com/31559413/189047619-dddebbb5-81a0-4dd2-9b56-63adbf04ed2b.png">

Reply:
<img width="960" alt="image" src="https://user-images.githubusercontent.com/31559413/189049638-35b47fa9-fe1c-4753-8a8a-ca5cbc98ed51.png">